server/pulp/server/util.py

```diff
@@ -258,6 +258,8 @@ def copytree(src, dst, symlinks=False, ignore=None):
             else:
                 # Don't need to copy attributes
                 copy(srcname, dstname)
+                st = os.stat(srcname)
+                os.utime(dstname, (st.st_atime, st.st_mtime))
```

Comment:
afaik we don't want to copy over the selinux labels (otherwise we could use the shutil.copytree function)